### PR TITLE
Enabling dev_dbg() logs support for JP5 & JP6

### PR DESCRIPTION
**Pull request by Arun-Prasad-V**
Tracked on LRS-1174

### CODE REVIEW

File: build_all.sh [hunk 1]
```diff
@@ -69,6 +75,9 @@ else
 #jp4/5
     cd $SRCS/$KERNEL_DIR
     make ARCH=arm64 O=$TEGRA_KERNEL_OUT tegra_defconfig
+    if [[ "$DEVDBG" == "1" ]]; then
+        scripts/config --file $TEGRA_KERNEL_OUT/.config --enable DYNAMIC_DEBUG
```
Comment (Nir-Az) [inline build_all.sh:79]: What will happen for JP4 here? (not mandatory as we stop support but better not to break it too fast)

Comment (Arun-Prasad-V) [reply build_all.sh:79]: It won't break JP4, as this flag anyway present in JP4 and enabled by default.

File: build_all.sh [hunk 2]
```diff
@@ -3,11 +3,17 @@
 set -e
 
 if [[ "$1" == "-h" ]]; then
-    echo "build_all.sh [JetPack_version] [JetPack_Linux_source]"
+    echo "build_all.sh [--no-dev-dbg] [JetPack_version] [JetPack_Linux_source]"
```
Comment (Nir-Az) [inline build_all.sh:6]: Did you try building it on Jenkins? JP5 + JP6?

File: build_all.sh [hunk 3]
```diff
@@ -42,7 +48,20 @@ if [[ "$JETPACK_VERSION" == "6.0" ]]; then
     cd $SRCS
     export KERNEL_HEADERS=$SRCS/kernel/kernel-jammy-src
     ln -sf $TEGRA_KERNEL_OUT $SRCS/out
-    make ARCH=arm64 -C kernel
+    if [[ "$DEVDBG" == "1" ]]; then
+        cd $KERNEL_HEADERS
+        make ARCH=arm64 defconfig
```
Comment (Nir-Az) [inline build_all.sh:53]: Can you explain a bit about all the added lines?
Thanks :)

Comment (Nir-Az) [reply build_all.sh:53]: Maybe add some comments?

Comment (Arun-Prasad-V) [reply build_all.sh:54]: Added comments.

File: build_all.sh (hunk 2 above)
Comment (Arun-Prasad-V) [reply build_all.sh:6]: Yes, built and verified the output Images for JP5.0.2 and JP6. I'm able to enable dev_dbg() logs from d4xx.c file.

**Review by Nir-Az — APPROVED** [review]
Ready for merge?

**Comment by Arun-Prasad-V** [comment]
> Ready for merge?

Yes, @Nir-Az.

**Pull request merged**